fdb-extensions/src/main/java/com/apple/foundationdb/async/RankedSet.java

```diff
@@ -143,10 +190,7 @@ public RankedSet(Subspace subspace, Executor executor) {
      */
     public CompletableFuture<Boolean> add(TransactionContext tc, byte[] key) {
         checkKey(key);
-        // Use the hash of the key, instead a p value and randomLevel. The key is likely Tuple-encoded.
```

Comment:
While the other lines in this comment are needed any more, I think this first line is still valuable.

Reply:
Hmm, okay. Restored.